### DISCUSSION TITLE
Remove promoted hashtags from twitter

### DISCUSSION THREAD

**Discussion by REVENGE977** [ReVanced/revanced-discussions#412] [Question]
suggestion: a patch for removing promoted hashtags on twitter would be cool.

**Comment by oSumAtrIX** [comment]
What do you mean?

**Reply by Sculas** [reply]
I guess this?
![Screenshot_0b2fce7a16bf2b728d6ffa28c8d60efb](https://user-images.githubusercontent.com/22832313/182047510-a366dd41-d8d8-4cbe-98a3-a76cbadbb2a4.jpg)

**Reply by oSumAtrIX** [reply]
Sounds useless as this does not take up any space or disturb you.

**Comment by REVENGE977** [comment]
like this

![image](https://user-images.githubusercontent.com/32039975/182061074-5cfb8142-9eaf-4eed-9edb-a507e4ff5b33.png)
